README.md

```diff
@@ -12,7 +12,37 @@ Run through those steps **including the addition of `/etc/hosts` aliases and the
 
 ### Configure xPro and Open edX
 
-See [Configure Open edX](docs/configure_open_edx.md)
+1. See [Configure Open edX](docs/configure_open_edx.md)
+1. Add an alias to `/etc/hosts` for Open edX. We have standardized this alias
+  to `edx.odl.local`. Your `/etc/hosts` entry should look like this:
+    
+    ```
+    127.0.0.1       edx.odl.local
+    ```
+
+### Add settings values
+
+Add the following settings in your `.env` file:
+
+```
+MAILGUN_RECIPIENT_OVERRIDE=<your email address>
+
+# Ask a fellow developer for these values 
+MAILGUN_SENDER_DOMAIN=
+MAILGUN_URL=
+MAILGUN_KEY=
+```
+
+OS-specific settings to add to your `.env` file:
+
+```
+# Linux
+OPENEDX_API_BASE_URL=http://edx.odl.local:18000
```

Comment:
I'm confused. That IP should always be the home IP address, shouldn't it? If so, from what I understand it's not necessary to use the `/etc/hosts` alias, but it should still work.

Reply:
I was getting this error from the worker because the container doesn't have that in its `/etc/hosts`, changing to the edx IP (e.g. `http://172.22.0.1:18000/`) fixed it:
```
celery_1  | urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='edx.odl.local', port=18000): Max retries exceeded with url: /user_api/v1/account/registration/ (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f0c62a035f8>: Failed to establish a new connection: [Errno -2] Name or service not known',))
celery_1  | 
celery_1  | During handling of the above exception, another exception occurred:
celery_1  | 
celery_1  | Traceback (most recent call last):
celery_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 382, in trace_task
celery_1  |     R = retval = fun(*args, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/celery/app/trace.py", line 641, in __protected_call__
celery_1  |     return self.run(*args, **kwargs)
celery_1  |   File "/src/courseware/tasks.py", line 12, in create_lms_user_from_id
celery_1  |     create_lms_user(user)
celery_1  |   File "/src/courseware/api.py", line 43, in create_lms_user
celery_1  |     **OPENEDX_REQUEST_DEFAULTS,
celery_1  |   File "/usr/local/lib/python3.6/site-packages/requests/api.py", line 116, in post
celery_1  |     return request('post', url, data=data, json=json, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/requests/api.py", line 60, in request
celery_1  |     return session.request(method=method, url=url, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/requests/sessions.py", line 533, in request
celery_1  |     resp = self.send(prep, **send_kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/raven/breadcrumbs.py", line 341, in send
celery_1  |     resp = real_send(self, request, *args, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/requests/sessions.py", line 646, in send
celery_1  |     r = adapter.send(request, **kwargs)
celery_1  |   File "/usr/local/lib/python3.6/site-packages/requests/adapters.py", line 516, in send
celery_1  |     raise ConnectionError(e, request=request)
```